modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -145,20 +144,20 @@ Amazon DynamoDB Connector v1.3
 === Batch Delete Item
 `<dynamodb:batch-delete-item>`
 
-
-FIXME: Add docs.
-
++++
+ The BatchDeleteItem operation deletes multiple items in one or more tables.   <a href="https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html">API Reference</a> 
```

Comment:
For this and all operations descriptions, you need to add some text before the link and a period after it. I suggest this. 

The BatchDeleteItem operation deletes multiple items in one or more tables.  For information, see the Amazon  DynamoDB <a href="https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html">API Reference</a>.

Please include Amazon DynamoDB in the link, so it is Amazon DynamoDB API Reference